packages/demo/yarn.lock

```diff
@@ -3118,8 +3131,18 @@
     "@walletconnect/relay-auth" "^1.0.4"
     "@walletconnect/safe-json" "^1.0.2"
     "@walletconnect/time" "^1.0.2"
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
looks like some merge conflicts need to be resolved in yarn.lock

Reply:
alright that is fixed now. Thanks for flagging!